quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/DockerKeycloakDistribution.java

```diff
@@ -90,9 +103,23 @@ public CLIResult run(List<String> arguments) {
             LOGGER.warn("Failed to start Keycloak container", cause);
         }
 
+        trySetRestAssuredPort();
+
         return CLIResult.create(getOutputStream(), getErrorStream(), getExitCode());
     }
 
+    private void trySetRestAssuredPort() {
+        try {
+            ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+            Class<?> restAssured = classLoader.loadClass("io.restassured.RestAssured");
+            Field port = restAssured.getDeclaredField("port");
+            port.set(null, keycloakContainer.getMappedPort(8080));
+        } catch (Exception ignore) {
+            // keeping the workaround to set the container port to restassured
```

Comment:
@vmuzikar The module no longer depends on `restassured`. This code is kept as is but we should improve later to properly expose ports as a follow up.